src/inference/optimize.js

```diff
@@ -26,7 +26,25 @@ module.exports = function(env) {
     EUBO: require('./eubo')(env)
   };
 
-  function Optimize(s, k, a, wpplFn, options) {
+  function Optimize(s, k, a, fnOrOptions, maybeOptions) {
+    var wpplFn, options;
```

Comment:
Is there a reason you renamed `wpplFn` to `model` in `Infer` but kept it here? If not, rename to `model` here as well?